db/seeds/book_author.seeds.rb

```diff
@@ -0,0 +1,33 @@
+SEEDS_COUNT = 50
+BOOK_DIMENSIONS_RANGE = (1.0..10.0).freeze
+
+SEEDS_COUNT.times do |index|
+  Author.create! do |author|
+    author.first_name = FFaker::Name.unique.first_name
+    author.last_name  = FFaker::Name.last_name
+  end
+
+  Book.create! do |book|
```

Comment:
You can do it in 1 line `FactoryBot.create_list(:book, 50, with_images: true, images_count: 4)`